bindings/rust/s2n-tls/src/connection.rs

```diff
@@ -809,6 +809,27 @@ impl Connection {
             hash_alg => Some(hash_alg.try_into()?),
         })
     }
+
+    pub fn tls_exporter(
+        &self,
+        label: &[u8],
```

Comment:
Note: this is passed as `&str` in Rust OpenSSL bindings (https://docs.rs/openssl/latest/openssl/ssl/struct.SslRef.html#method.export_keying_material) and the parameter order is a bit different. Happy to match that, though it does slightly constrain the API. Might make it less likely to swap parameter order unintentionally.

Reply:
We don't really try to maintain any parity with openssl so I think it's fine as is. I'd rather match the RFC than anything (which you have here).

https://datatracker.ietf.org/doc/html/rfc8446#section-7.5

```
   The exporter value is computed as:

   TLS-Exporter(label, context_value, key_length) =
       HKDF-Expand-Label(Derive-Secret(Secret, label, ""),
                         "exporter", Hash(context_value), key_length)
```